comments/models.py

```diff
@@ -112,33 +112,38 @@ def __str__(self):
         return self.text
 
 
-class CommentModeration(models.Model):
-    class CommentModerationState(models.TextChoices):
+class CommunityCommentModeration(models.Model):
+    class State(models.TextChoices):
         UNMODERATED = "UNMODERATED", "Unmoderated"
         APPROVED = "APPROVED", "Approved"
         REJECTED = "REJECTED", "Rejected"
         UNSURE = "UNSURE", "Unsure"
 
-    state = models.CharField(max_length=255, choices=CommentModerationState.choices, default=CommentModerationState.UNMODERATED)
+    state = models.CharField(max_length=255, choices=State.choices, default=State.UNMODERATED)
     comment = models.OneToOneField(
         to='django_comments_xtd.XtdComment', related_name='comment_moderation', on_delete=models.CASCADE)
 
     def __str__(self):
         return f'{self.comment.id} | {self.state}'
 
+    class Meta:
+        permissions = (
+            ("can_community_moderate", "Can moderate comments on PUBLIC SITE"),
```

Comment:
I know this is what @cbunicef suggested, but I'd like the all-caps ('PUBLIC SITE') to be changed to lower-case. I don't want to start a trend, and I feel it could be considered a bit patronising.